src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -38,12 +39,28 @@ public InputFileResolver(IFileSystem fileSystem)
         /// </summary>
         public ProjectInfo ResolveInput(string currentDirectory, string projectName)
         {
-            string projectFile = ScanProjectFile(currentDirectory);
+            var projectFile = ScanProjectFile(currentDirectory);
             var currentProjectInfo = ReadProjectFile(projectFile, projectName);
             var projectReferencePath = FilePathUtils.ConvertPathSeparators(currentProjectInfo.ProjectReference);
+            
             var projectUnderTestPath = Path.GetDirectoryName(Path.GetFullPath(Path.Combine(currentDirectory, projectReferencePath)));
-            var projectUnderTestInfo = FindProjectUnderTestAssemblyName(Path.GetFullPath(Path.Combine(projectUnderTestPath, Path.GetFileName(projectReferencePath))));
-            var inputFiles = FindInputFiles(projectUnderTestPath);
+            var projectReference = Path.Combine(projectUnderTestPath, Path.GetFileName(projectReferencePath));
+            var projectFilePath = Path.GetFullPath(projectReference);
+            var projectUnderTestInfo = FindProjectUnderTestAssemblyName(projectFilePath);
+            var paths = FindAllPaths(projectReference);
+            FolderComposite inputFiles = null;
+            for (var i = 0; i < paths.Count; i++)
+            {
+                _logger.LogDebug($"Scanning {paths[i]}");
+                if (inputFiles == null)
+                {
+                    inputFiles = FindInputFiles(paths[i]);
+                }
+                else
+                {
+                    inputFiles.Add(FindInputFiles(paths[i]));
```

Comment:
By adding the shared project input files to the root FolderComposite in the console report it looks like the project is a subfolder of the main project. Maybe it would be nicer if we could find a way it shows it as an extra project (which it is)

So not:
```
- \ExampleProject
--- Some mutant
--- \ExampleSharedProject
```
But
```
- \ExampleProject
--- Some mutant
- \ExampleSharedProject
```

We could do this by creating a "root" FolderComposite and add all projects to it. In the reporter the root folder should not be displayed since it does not exist.

Reply:
Implemented. I haven't tested reporting. But the root has no name.